src/connection/passwordless/actions.js

```diff
@@ -12,7 +12,8 @@ import {
   send,
   setPasswordlessStarted,
   setResendFailed,
-  setResendSuccess
+  setResendSuccess,
+  toggleTermsAcceptance as switchTermsAcceptance
```

Comment:
Would `switchTermsAcceptance` be better named as `toggleTermsAcceptanceParent` or similar? It looks like you're overriding that method here but the name suggests it does something different ... and even then, if it did, the name is pretty similar.

Reply:
renamed to `internalToggleTermsAcceptance`